lib/tax_cloud/address.rb

```diff
@@ -19,15 +19,17 @@ class Address < Record
     # Verify this address.
     #
     # Returns a verified TaxCloud::Address.
-    def verify
+    def verify(options = {})
+      with_rdi = options.fetch(:with_rdi, false)
+
       params = to_hash.downcase_keys
       if TaxCloud.configuration.usps_username
         params = params.merge(
           'uspsUserID' => TaxCloud.configuration.usps_username
         )
       end
       response = TaxCloud.client.request(:verify_address, params)
-      TaxCloud::Responses::VerifyAddress.parse(response)
+      TaxCloud::Responses::VerifyAddress.parse(response, with_rdi)
```

Comment:
Should we pass `options` through as is?

Reply:
```
?xml version="1.0" encoding="utf-8"?>
<soap:Envelope
	xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/"
	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
	xmlns:xsd="http://www.w3.org/2001/XMLSchema">
	<soap:Body>
		<VerifyAddressResponse
			xmlns="http://taxcloud.net">
			<VerifyAddressResult>
				<Address1>888 6th Ave</Address1>
				<City>New York</City>
				<State>NY</State>
				<Zip5>10001</Zip5>
				<Zip4>3502</Zip4>
				<rdi>Commercial</rdi>
				<ErrNumber>0</ErrNumber>
			</VerifyAddressResult>
		</VerifyAddressResponse>
	</soap:Body>
</soap:Envelope>
```

this is how it looks now. I'm not sure it should be added to the Address since it's more about address validation. Also, I'm unsure about backward compatibility if we add this to the Address record. So I decided to make it optional

Reply:
of course adding it to the Address would be the easiest way, but I'm not sure what `rdi` stands for :) My first intention was to fix the bug

Reply:
I didn't analyze the gem deeply, but if you sure adding `rdi` to `Address` is going to keep backward compatibility, I'll simply add a new accessor to `Address`

Reply:
https://github.com/txcrb/tax_cloud/pull/57/files all tests are passing but we need to make sure we don't break anything already existing

Reply:
I'll leave another ask on backwards compat.